src/components/organelle-wrapper.tsx

```diff
@@ -0,0 +1,101 @@
+import * as React from 'react';
+declare var Organelle: any;
+
+interface OrganelleWrapper {
+  name: string,
+  modelProperties: any,
+  showBinding: boolean,
+  doAddHormone: boolean,
+  addEnzyme: boolean,
+  setActiveAssay: Function,
+  currentView: any,
+  setGraphState: Function
+}
+
+class OrganelleWrapper extends React.Component<any, any> {
+    model: any;
+  constructor(props: OrganelleWrapper) {
+    super(props);
+    this.model = null;
+    this.addHormone = this.addHormone.bind(this);
+  }
+
+  componentDidMount() {
+    const {modelProperties} = this.props;
+
+    Organelle.createModel({
+      element: this.props.name,
+      background: {
+        file: "assets/melanocyte.svg",
+        selector: "#cell"
+      },
+      properties: modelProperties,
+      calculatedProperties: {},
+      clickHandlers: [
+        {
+          selector: "#melanocyte_x5F_cell",
+          action: this.organelleClick.bind(this, "cytoplasm")
+        },
+        {
+          selector: "#nucleus_x5F_A",
+          action: this.organelleClick.bind(this, "nucleus")
+        },
+      ],
+      species: [
+        "organelles/hexagon.yml",
+        "organelles/triangle.yml",
+        "organelles/g-protein.yml",
+        "organelles/g-protein-part.yml"
+      ]
+    }).then((m:any) => {
+      this.model = m;
+
+      this.model.setTimeout(() => {
+        let cell:any = document.querySelector(`#${this.props.name} #cellshape_0_Layer0_0_FILL`)
```

Comment:
I had to wrap all the query selectors like this, for two reasons: first, the query might come back null, which makes TS unhappy. Second, the type returned by the selector doesn't seem to have a `style` property by default, so I had to cast it as `any` to avoid TS complaints.